File: contest_1878_1.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int t;
    cin>>t;
    while(t--)
    {
        int ln,num,found=0;
        cin>>ln>>num;
        for(int i=0;i<ln;i++)
        {
            int temp;
            cin>>temp;
            if(temp==num)
                found=1;
                
        }
        if(found)
            cout<<"YES"<<endl;
        else
            cout<<"NO"<<endl;
    }

}

File: contest_1878_1.py
def main():
    t=int(input())
    while(t):
        t-=1
        found=0
        ln,num=map(int,input().split())
        temp=map(int,input().split())
        for i in temp:
            if(i==num):
                found=1
        if found:
            print("YES")
        else:
            print("NO")
main()